### DISCUSSION TITLE
Can't seem to write into Documents folder

### DISCUSSION THREAD

**Discussion by yakman2020** [actions/virtual-environments#9500] [Image configuration]
The Imaging Source IC Imaging SDK (3.5) for windows only installs to the documents folder and has no control to install to anywhere else. I have a github actions setup taks trying to download and install the SDK which is running the installed, but the results don't show up in the folder.  I'm stuck having to keep a copy of the SDK as a gzipped tar in the repo which seems inelegant and fragile. 

Is there some way to enable writing to the ~/Documents folder (c:/user/runneradmin/Documents) rather than failing silently?

**Comment by mikhailkoliada** [comment]
Hey!

Not sure what you action does under the hood, but plain test:

```  build:
    strategy:
      matrix: 
        os: ["windows-2019", "windows-2022"]
    runs-on: ${{ matrix.os }}
    steps:
      - name: test creation
        run: |
         New-Item -Type File -Name test.txt -Path ~/Documents
         Get-ChildItem ~/Documents

```
shows nothing changed so far and you can put your data there :)
